src/edit/DistortableImage.EditToolbar.js

```diff
@@ -144,17 +144,21 @@ var EditOverlayAction = LeafletToolbar.ToolbarAction.extend({
   }
   });
 
-L.DistortableImage.EditToolbar = LeafletToolbar.Popup.extend({
-	options: {
-		actions: [
-			ToggleTransparency,
-			RemoveOverlay,
-			ToggleOutline,
-			ToggleEditable,
-			ToggleRotateDistort,
-			ToggleExport,
-      EnableEXIF,
-			ToggleOrder
-		]
-	}
-});
+	setTimeout(function(){
+		var toolbarStyle = L.DistortableImage._options.toolbarType;
```

Comment:
Oh, shall we say maybe:

`var toolbarStyle = L.DistortableImage._options.toolbarType || "Popup";`

and add to the documentation this default and the new options parameter? Thank you so much!!!